stories/layout/report.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React from 'react'
+import { Grid, Item, Layout } from '../../src'
+import { Box, loremIpsum } from '../core'
+import { colors1, colors2 } from '../core/stories.css'
+import styles from './layout.css'
+
+function P(props) {
+  return (
+    <Item size={12}>
+      <p {...props} />
+    </Item>
+  )
+}
+
+function Card({
+  children,
+  height = 70,
+}: {
+  children?: any,
+  height?: number,
+}): React$Element<any> {
+  return (
+    <Item size={12} style={{ height }}>
+      <div style={{ height: '100%' }} className={styles.card}>
+        <P>{children}</P>
+      </div>
+    </Item>
+  )
+}
+
+export default function() {
+  return (
+    <Layout
+      type="parent"
+      className={`${styles.page} ${styles.report} ${styles.hasReportSubheader}`}
+    >
+      <Layout fixed="top">
+        <Layout className={styles.header}>
+          <Grid root>
+            <h1>Header</h1>
+          </Grid>
+        </Layout>
+        <Layout className={styles.subheader} margin="none">
+          <Grid root margin="horizontal">
+            <Item size={12}><h2>Subheader</h2></Item>
+            <Box size={2} type="C" />
+            <Box size={2} type="C" />
+            <Box size={2} type="C" />
+            <Box size={2} type="C" />
+            <Box size={2} offset={2} type="C" />
+          </Grid>
+        </Layout>
+      </Layout>
+      <Layout type="parent" className={`${styles.main} ${colors2}`}>
+        <Layout type="stretch">
+          <Grid root>
+            <Grid align="stretch">
+              <Grid size={1} className={styles.nav} align="top">
+                <Item size={12} />
```

Comment:
This is really hacky, basically adds an element with 0 height so we can use it's margin bottom as margin-top for the next element. We may need an option like `includeMarginTop` but that also sounds really weird